fiftyone/brain/internal/core/redaction.py

```
@@ -0,0 +1,426 @@
+        self.label_field = label_field
+        self.label_classes = label_classes
+        self.redaction_type = redaction_type
+        self.redaction_method = redaction_method
+        self.redaction_field = redaction_field
+        self.validate_label_classes()
+        self.make_redaction_field()
+
+    def validate_label_classes(self):
+        try:
+            self.label_classes = list(map(str.strip, self.label_classes))
+        except:
```

Ruff (0.14.11)

Do not use bare `except`

(E722)